examples/spring-web/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+logging:
+  level:
+    root: info
+spring:
+  security:
+    oauth2:
+      client:
+        registration:
+          myApp:
+            client-id: https://inrupt.github.io/solid-client-java/clients/springboot-testing.jsonld
+            provider: inrupt
+            scope:
+              - openid
+              - webid
+            client-authentication-method: none
+            authorization-grant-type: authorization_code
```

Comment:
Should we add a comment that this value should match the grant types in the Client ID document?

Reply:
I think authorization_code is a baseline assumption, so I don't see a need to add a comment in particular